src/main/java/com/nvidia/spark/rapids/jni/DecimalUtils.java

```diff
@@ -56,7 +56,49 @@ public static Table divide128(ColumnView a, ColumnView b, int quotientScale) {
     return new Table(divide128(a.getNativeView(), b.getNativeView(), quotientScale));
   }
 
+  /**
+   * Subtract two DECIMAL128 columns and produce a DECIMAL128 result rounded to the specified
+   * scale with overflow detection.
+   *
+   * NOTE: This is very specific to Spark 3.4. This method won't work with previous versions
+   * of Spark. We don't need this for versions prior to Spark 3.4
+   *
+   * @param a input, must match row count of the other input
+   * @param b input, must match row count of the other input
+   * @param targetScale scale to use for the result
+   * @return table containing a boolean column and a DECIMAL128 result column of the specified
+   *         scale. The boolean value will be true if an overflow was detected for that row's
+   *         DECIMAL128 result value. A null input row will result in a corresponding null output
+   *         row.
+   */
+
+  public static Table subtract128(ColumnView a, ColumnView b, int targetScale) {
+    return new Table(subtract128(a.getNativeView(), b.getNativeView(), targetScale));
+  }
+  /**
+   * Add two DECIMAL128 columns and produce a DECIMAL128 result rounded to the specified
+   * scale with overflow detection.
+   *
+   * NOTE: This is very specific to Spark 3.4. This method won't work with previous versions
```

Comment:
nit: can we reword this to say that this add is not compatible with versions of spark prior to 3.4 instead of it "won't work" with them?